tests/simplex/conv_hex_to_simplex.cc

```diff
@@ -99,6 +106,18 @@ check_file() // for dim = spaceim
     grid_out.write_vtk(tria, deallog.get_file_stream());
   };
 
+  // Temporary: See results in paraview
+  std::ofstream out1("Quad.vtk");
+  GridOut       grid_out1;
+  grid_out1.write_vtk(in_tria, out1);
```

Comment:
This doesn't show up in the test output, and so if someone breaks the functionality, we will not learn about it. You'll have to put the VTK output into the `.output` file in some form.